corehq/apps/reports/views.py

```diff
@@ -1059,7 +1059,7 @@ def _product_name(product_id):
                 (_product_name(product_id), entry_map[product_id])
                 for product_id in entry_map
             ]
-            product_tuples.sort(key=lambda product_name, entry_state: product_name)
+            product_tuples.sort(key=lambda x: x[0])
```

Comment:
it now receives a list of arguments?
do we need to check for this in more places?